vms/platformvm/addrstate/camino_address_state.go

```diff
@@ -9,38 +9,39 @@ type (
 const (
 	// Bits
 
-	AddressStateBitRoleAdmin AddressStateBit = 0 // super role
-
-	AddressStateBitRoleKYCAdmin                AddressStateBit = 1 // allows to set KYCVerified and KYCExpired
-	AddressStateBitRoleOffersAdmin             AddressStateBit = 2 // allows to set OffersCreator
-	AddressStateBitRoleConsortiumAdminProposer AddressStateBit = 3 // allows to create admin add/exclude member proposals
-
-	AddressStateBitKYCVerified    AddressStateBit = 32
-	AddressStateBitKYCExpired     AddressStateBit = 33
-	AddressStateBitConsortium     AddressStateBit = 38
-	AddressStateBitNodeDeferred   AddressStateBit = 39
-	AddressStateBitOffersCreator  AddressStateBit = 50
-	AddressStateBitCaminoProposer AddressStateBit = 51
+	AddressStateBitRoleAdmin               AddressStateBit = 0 // allow to set role bits
+	AddressStateBitRoleKYCAdmin            AddressStateBit = 1 // allows to set KYCVerified and KYCExpired
+	AddressStateBitRoleOffersAdmin         AddressStateBit = 2 // allows to set OffersCreator
+	AddressStateBitRoleConsortiumSecretary AddressStateBit = 3 // allows to create admin add/exclude member proposals
```

Comment:
> // allows to create admin add/exclude member proposals

Comment is unclear to me. Does this bit allows to create member _and admin_ add/exclude proposals? Or only member add/exclude proposals?

Reply:
Add/exclude member proposals can be created by any of consortium members.
But admin add/exclude member proposals can be created only by ConsortiumSecretary

Reply:
The expression of "admin add/exclude member" is not clear to me. 

Does it mean that ConsortiumMember can create proposals to _flag_ an address with bits `0`, `1`, and `4`? (and maybe `3`)

Reply:
I rearranged comment a bit. Its now clear that its about admin-proposals, which is referring to specific proposal type as well as proposal names.